gauges/backends.go

```diff
@@ -31,6 +31,18 @@ func (g *Gauges) MaxBackends() prometheus.Gauge {
 	)
 }
 
+// CurrentBackends returns the number of backends currently connected to all databases 
+func (g *Gauges) CurrentBackends() prometheus.Gauge {
+	return g.new(
+		prometheus.GaugeOpts{
+			Name:        "postgresql_current_backends",
+			Help:        "Current number of concurrent connections in all databases",
+			ConstLabels: g.labels,
+		},
+		"SELECT sum(numbackends) FROM pg_stat_database;",
```

Comment:
Is it possible to export this metric by database with a label? Than we can aggregate it later as needed

Reply:
This is already done with `postgresql_backends_total` metric (some line of codes above). 

This one is for the instance as a whole. It's not the best approach but for now can solve the problem with a instance with a lot of databases.

Reply:
Hmm we could solve this by adding a `instance` label but I think this information is not available ATM :/

Reply:
Yeap, we've to change the config file structure to allow grouping databases as you suggest. BTW,  I mentioned it on the PR description to be visible. Maybe a generous contributor can speed it up to us.